text-ui-test/EXPECTED.TXT

```diff
@@ -16,6 +16,7 @@ edit -p PARTICIPANT -n NUMBER -email EMAIL -e EVENT: Edit participant contact in
 edit -e EVENT -name EVENT_NAME -t TIME -v VENUE -u PRIORITY: Edit event info.
 mark -e EVENT -s STATUS: Mark an event as done or not done.
 mark -p PARTICIPANT -e EVENT -s STATUS: Mark a participant as present or absent.
+mark -m ITEM -e EVENT -s STATUS: Mark an item as accounted or unaccounted.
```

Comment:
Perhaps try to add a command about marking item in 'input.txt' for testing the MarkItemCommand.

Reply:
Noted, I'll add them accordingly.